modules/core/src/main/java/org/apache/ignite/internal/jdbc/thin/JdbcThinTcpIo.java

```diff
@@ -102,8 +102,11 @@ public class JdbcThinTcpIo {
     /** Version 2.13.0. */
     private static final ClientListenerProtocolVersion VER_2_13_0 = ClientListenerProtocolVersion.create(2, 13, 0);
 
+    /** Version 2.17.0. Removes nested transaction support. */
+    private static final ClientListenerProtocolVersion VER_2_17_0 = ClientListenerProtocolVersion.create(2, 17, 0);
```

Comment:
Do we still need new version of protocol?

Reply:
Agree. We don't need it since we haven't actually changed it